<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgetPassword_1</class>
 <widget class="QWidget" name="ForgetPassword_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ForgetPassword_1{
border-radius:10px;
}</string>
  </property>
  <widget class="QPushButton" name="btn_MinR">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>17</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btn_MinR
{
     border-image:url(:/image/Min01.png);
}
QPushButton:hover#btn_MinR
{
     border-image:url(:/image/Min_hover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_picLoveR">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>26</y>
     <width>220</width>
     <height>55</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>55</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/life_ring2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_SubmitF1">
   <property name="geometry">
    <rect>
     <x>509</x>
     <y>390</y>
     <width>317</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-size:24px;
}
QPushButton:hover#btn_SubmitF1{
background-color:#A9E2F3;
color: rgb(254, 245, 237);
border-radius:5px;
}
QPushButton#btn_SubmitF1{
background-color:#58ACFA;
color: rgb(254, 245, 237);
border-radius:5px;
}





</string>
   </property>
   <property name="text">
    <string>提交申请</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_CloseR">
   <property name="geometry">
    <rect>
     <x>858</x>
     <y>17</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btn_CloseR
{
      border-image: url(:/image/Close01.png);
}
QPushButton:hover#btn_CloseR
{
      border-image: url(:/image/Close_hover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEdit_oldpassword">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>310</y>
     <width>317</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	font-size:20px;
}
QLineEdit#lineEdit_oldpassword{
border:1px solid rgb(211,211,211);
}
QLineEdit:hover#lineEdit_oldpassword{
border:1px solid rgb(255,170,0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEdit_Email">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>190</y>
     <width>317</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	font-size:20px;
}
QLineEdit#lineEdit_Email{
border:1px solid rgb(211,211,211);
}
QLineEdit:hover#lineEdit_Email{
border:1px solid rgb(255,170,0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2R">
   <property name="geometry">
    <rect>
     <x>518</x>
     <y>90</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:27px;
color: rgb(110, 110, 110);</string>
   </property>
   <property name="text">
    <string>忘记密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_errRF1">
   <property name="geometry">
    <rect>
     <x>518</x>
     <y>138</y>
     <width>250</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:16px;
color: rgb(251, 173, 88);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEdit_Password_new">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>230</y>
     <width>317</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	font-size:20px;
}
QLineEdit#lineEdit_Password_new{
border:1px solid rgb(211,211,211);
}
QLineEdit:hover#lineEdit_Password_new{
border:1px solid rgb(255,170,0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEdit_Password_check">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>270</y>
     <width>317</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	font-size:20px;
}
QLineEdit#lineEdit_Password_check{
border:1px solid rgb(211,211,211);
}
QLineEdit:hover#lineEdit_Password_check{
border:1px solid rgb(255,170,0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Change1">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>230</y>
     <width>40</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Change2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>270</y>
     <width>40</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>177</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:bold &quot;Agency FB&quot;;
font-size:18px;
color: rgb(110, 110, 110);</string>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>267</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:bold &quot;Agency FB&quot;;
font-size:18px;
color: rgb(110, 110, 110);</string>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>312</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:bold &quot;Agency FB&quot;;
font-size:18px;
color: rgb(110, 110, 110);</string>
   </property>
   <property name="text">
    <string>旧密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>222</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:bold &quot;Agency FB&quot;;
font-size:18px;
color: rgb(110, 110, 110);</string>
   </property>
   <property name="text">
    <string>新密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>btn_MinR</zorder>
  <zorder>label_picLoveR</zorder>
  <zorder>btn_SubmitF1</zorder>
  <zorder>btn_CloseR</zorder>
  <zorder>lineEdit_oldpassword</zorder>
  <zorder>lineEdit_Email</zorder>
  <zorder>label_2R</zorder>
  <zorder>label_errRF1</zorder>
  <zorder>lineEdit_Password_new</zorder>
  <zorder>lineEdit_Password_check</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>label_4</zorder>
  <zorder>btn_Change1</zorder>
  <zorder>btn_Change2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_Email</tabstop>
  <tabstop>lineEdit_Password_new</tabstop>
  <tabstop>lineEdit_Password_check</tabstop>
  <tabstop>lineEdit_oldpassword</tabstop>
  <tabstop>btn_SubmitF1</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
